src/components/Books.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = () => (
+  <>
+    <ListBooks />
+    <AddBooks />
+  </>
+);
+
+const ListBooks = () => {
+  const booksData = [{
+    id: 1,
+    name: 'The Hunger Games',
+    author: 'Suzanne Collins',
+  }, {
+    id: 2,
+    name: 'Dune',
+    author: 'Frank Herbert',
+  }, {
+    id: 3,
+    name: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+  }];
+
+  return (
+    <ul className="books_list">
+      {booksData.map((data) => (
+        <li key={data.id}>
+          <Book name={data.name} author={data.author} />
+        </li>
+      ))}
+    </ul>
+  );
+};
```

Comment:
- [ ] While this piece of code is working, I would still recommend putting it into a separate file, as a different component, as you may want to use it more, so that would make the imports a bit easier and will also keep your file a bit more organized. So please consider moving this piece to a new file.